exo-run/src/service-runner.ls

```diff
@@ -39,7 +39,7 @@ class ServiceRunner extends EventEmitter
         ..on 'online', -> done?!
         ..on 'error', (message) ~> @emit 'error', error-message: message
 
-    @watcher = watch @config.root, ignore-initial: yes, ignored: /.*\/node_modules\/.*/
+    @watcher = watch @config.root, ignore-initial: yes, ignored: [/.*\/node_modules\/.*/, /(^|[\/\\])\../]
```

Comment:
I don't understand this regex. Can you please shed some light on it, ideally with a comment that explains it?